src/Select.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { theme } from 'styled-system'
+
+import { Box, Flex, Icon, Label } from '../src'
+
+const parseChevronIcon = isExpanded => {
+  const ChevronIcon = isExpanded
+    ? Icon.extend`
+        transform: rotate(180deg);
+      `
+    : Icon
+
+  return <ChevronIcon name="chevronDown" size={12} color="black" />
+}
+
+const SelectFlex = Flex.extend`
+  cursor: pointer;
+  height: 48px;
+  border-radius: ${theme('radius')};
+  border-width: 1px;
+  border-style: solid;
+  border-color: ${theme('colors.borderGray')};
+`
+
+const SelectBox = Box.extend`
+  cursor: pointer;
+`
+
+const SelectLabel = Label.extend`
+  cursor: pointer;
```

Comment:
I'd avoid changing the Label styles here, if we want to change the default cursor here, we might want to change that across the board – either way that's probably something we wanna run by the design team

Reply:
I took a step back and review the pattern and UX. Agree that for now, we don't wanna set a cursor style on `Label` (essentially an HTML native `<label>` tag). Let me talk to @chrisdroukas & @odmln about if the `Label` should have a default cursor style for all, or should it not.